usb/usb_host_msc/src/msc_scsi_bot.c

```diff
@@ -290,7 +290,11 @@ static esp_err_t bot_execute_command(msc_device_t *device, msc_cbw_t *cbw, void
     MSC_RETURN_ON_ERROR( msc_bulk_transfer(device, (uint8_t *)cbw, CBW_SIZE, MSC_EP_OUT) );
 
     if (data) {
-        MSC_RETURN_ON_ERROR( msc_bulk_transfer(device, (uint8_t *)data, size, ep) );
+        if (size <= device->xfer->data_buffer_size) {
+            MSC_RETURN_ON_ERROR( msc_bulk_transfer(device, (uint8_t *)data, size, ep) );
```

Comment:
Is there any value is keeping the pass-by-copy version around anymore? Can we simply update `msc_bulk_transfer()` to be zero copy.

Reply:
Yes, we can will fix in next commit

Reply:
Fixed by 4a7cd3db29c44c5a4925737176084c831bb54f8a